packages/api/providers/RequestProvider.js

```diff
@@ -0,0 +1,12 @@
+const { ServiceProvider } = require('@adonisjs/fold');
+
+class MyServiceProvider extends ServiceProvider {
+	boot() {
+		const Request = use('Adonis/Src/Request');
+		Request.macro('has', (filters = {}, value = '') => Object.keys(filters).includes(value));
```

Comment:
Check if there is a param in a filters object (from `request.all()`)

Usage:
```js
const filters = request.all();

// instead of
if (!!filters.orders || filters.orders === '') {}

// now we can do
if (request.has(filters, 'orders')) {}
```